api/query/cache/index/index.go

```diff
@@ -31,6 +31,20 @@ var (
 	errZeroRun            = errors.New("Cannot ingest run with ID of 0")
 )
 
+// ErrRunExists returns the error associated with an attempt to perform
+// operations on a run currently unknown to an Index when the Index, in fact,
+// already knows about the run.
+func ErrRunExists() error {
```

Comment:
I don't know what is the convention in Go, but what is the benefit of exporting a function that returns the global variable over just exporting the global variable?

Reply:
It prevents other packages from executing `index.ErrRunExists = somethingElse`.